dashbuilder-client/dashbuilder-renderers/dashbuilder-renderer-default/src/main/java/org/dashbuilder/renderer/client/selector/SelectorLabelSetDisplayer.java

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dashbuilder.renderer.client.selector;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import org.dashbuilder.dataset.DataColumn;
+import org.dashbuilder.dataset.DataSetLookupConstraints;
+import org.dashbuilder.dataset.group.ColumnGroup;
+import org.dashbuilder.dataset.group.DataSetGroup;
+import org.dashbuilder.dataset.sort.DataSetSort;
+import org.dashbuilder.dataset.sort.SortOrder;
+import org.dashbuilder.displayer.ColumnSettings;
+import org.dashbuilder.displayer.DisplayerAttributeDef;
+import org.dashbuilder.displayer.DisplayerAttributeGroupDef;
+import org.dashbuilder.displayer.DisplayerConstraints;
+import org.dashbuilder.displayer.client.AbstractErraiDisplayer;
+import org.jboss.errai.ioc.client.container.SyncBeanManager;
+
+@Dependent
+public class SelectorLabelSetDisplayer extends AbstractErraiDisplayer<SelectorLabelSetDisplayer.View> {
```

Comment:
There is one potential UX issue with this new SelectorLabel displayer: it would be nice to have something similar to "X" button which we have for Multiple SelectorDropdown displayer which would clean all the selected labels and would bring the selector to its initial state (no labels selected). Otherwise user has to "unclick" each of the selected labels individually if he wants reset to initial state. Could you implement that, please?